src/App.jsx

```diff
@@ -10,23 +10,26 @@ function App() {
   return (
     <div className="App">
       <nav className="navBar">
-        <Link className="bookstoreTitle" to="/BookStoreReact_Vite/">
-          Geeky Book Store
-        </Link>
-        <div className="navLinks">
-          <Link className="link" to="/BookStoreReact_Vite/books">
-            Books
-          </Link>
-          <Link className="link" to="/BookStoreReact_Vite/categories">
-            Categories
-          </Link>
-          <Link className="link" to="/BookStoreReact_Vite/contact">
-            Contact
-          </Link>
+        <div className="fstdiv">
           <Link className="bookstoreTitle" to="/BookStoreReact_Vite/">
-            <img className="accountPic" src={accountPic} alt="" />
+            Bookstore CMS
           </Link>
+          {/* <div className="navLinks"> */}
+            <Link className="link" to="/BookStoreReact_Vite/books">
+              Books
+            </Link>
+            <Link className="link" to="/BookStoreReact_Vite/categories">
+              Categories
+            </Link>
+            <Link className="link" to="/BookStoreReact_Vite/contact">
+              Contact
+            </Link>
```

Comment:
- [ ] You have done extremely well in implementing the changes requested by the previous reviewers, however, there are still some styling you are missing, kindly pay close attention to the given design as requested by the previous reviewers. Matching your design to the given design requirements enhances the chances of creating a successful and well-rounded design solution that meets the needs of users, complies with regulations, and performs effectively and efficiently. :slightly_smiling_face: 

Below is an illustration of yours and the given Design, please pay attention to the red markings on your design: :arrow_down: 

<table>
    <tr>
      <td>Your Design</td>
      <td>Given Design</td>
    </tr>
    <tr>
      <td>

![Student DesS](https://github.com/GeekyHacks/BookStoreReact_Vite/assets/43528594/e703f3a2-5a89-4f3c-8607-e11d68ad4e5d)

</td>
      <td>

![Zeplin](https://github.com/GeekyHacks/BookStoreReact_Vite/assets/43528594/65f79b18-4e48-4e3b-8460-7b74be13fa43)

</td>
    </tr>
  </table>